apps/frontend/src/config.ts

```diff
@@ -1,8 +1,6 @@
 export const GTM_CONTAINER_ID = process.env.REACT_APP_GTM_CONTAINER_ID;
 export const API_BASE_URL =
   process.env.REACT_APP_API_BASE_URL || 'http://localhost:3333';
-export const ALGOLIA_APP_ID =
-  process.env.REACT_APP_ALGOLIA_APP_ID || 'LVYWOPQ0A9';
 export const ALGOLIA_INDEX =
   process.env.REACT_APP_ALGOLIA_INDEX || 'asap-hub_research_outputs_dev';
```

Comment:
is asap-hub_research_outputs_dev the correct name until we fully migrate over?

Reply:
yes